### DISCUSSION TITLE
sever protocol for Adonis v6

### DISCUSSION THREAD

**Discussion by azurha** [unpoly/unpoly#605] [Ideas]
Hello, 
Is a server protocol implementation for the Adonis v6, javascript backend framework, planned?
Thanks

**Comment by adam12** [comment]
Most of the server protocols outside of the Rails version have been community-initiated and driven.

The backend server protocol is optional and provided you have access to request and response headers inside your application, you can use the server protocol without any Unpoly-specific backend dependency.
